src/function/lazyLoading.js

```diff
@@ -7,13 +7,20 @@ function($) {
         if (!self.sprite && self.lasyEmoji[0]) {
             var pickerTop = self.picker.offset().top,
                 pickerBottom = pickerTop + self.picker.height() + 20;
-            self.lasyEmoji.each(function() {
+
+            self.picker.find(".lazy-emoji").each(function() {
```

Comment:
Note that if we loop over self.lasyEmoji, jquery migrate considers these to be 'disconnected form document' and issues a deprecation warning.

Instead I have had to use a live .find(), which may have a small performance hit, although I have not noticed any.
Added an optimisation which would help anyway